### PR TITLE
@tempnam in ExportService.php #4304

### LINKED ISSUE #4304: [Feature] @tempnam in ExportService.php
I don't know if this closing error prompt is necessary,But at least it had an impact on my troubleshooting of environmental errors,Please evaluate whether you need to remove the '@'.
be located ./app/Services/ExportService.php:170
`$zip_file = @tempnam('/tmp', 'zip');`

### PR DESCRIPTION
**Pull request by harshad389**
Closes #4304

How to test the feature manually:

1. Manually tested

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested
- [x] unit tests written (optional if too hard)
- [x] documentation updated

[Additional information can be found in the documentation](https://github.com/FreshRSS/FreshRSS/tree/edge/docs/en/developers/04_Pull_requests.md).

### CODE REVIEW

File: CREDITS.md [hunk 1]
```diff
@@ -176,3 +176,5 @@ People are sorted by name so please keep this order.
 * [xnaas](https://github.com/xnaas): [contributions](https://github.com/FreshRSS/FreshRSS/pulls?q=is:pr+author:xnaas), [Web](https://xnaas.info/)
 * [Yamakuni](https://github.com/Yamakuni): [contributions](https://github.com/FreshRSS/FreshRSS/pulls?q=is:pr+author:Yamakuni), [Web](https://ofanch.me/)
 * [yzqzss|一座桥在水上](https://github.com/yzqzss): [contributions](https://github.com/FreshRSS/FreshRSS/pulls?q=is:pr+author:yzqzss), [Web](https://blog.othing.xyz/)
+* [Harshad Hirapara](https://github.com/harshad389): [contributions](https://github.com/FreshRSS/FreshRSS/pulls?q=is:pr+author:harshad389)
```
Comment (Alkarex) [inline CREDITS.md]: Please keep alphabetic order

Comment (harshad389) [reply CREDITS.md]: Thanks you for suggetions. Let me correct this.

Comment (harshad389) [reply CREDITS.md]: I have updated it. Please review it?

**Comment by Alkarex** [comment]
Thanks!

**Pull request merged**